src/components/Picker/pickerStyles/index.android.js

```diff
@@ -5,7 +5,7 @@ const pickerStyles = isDisabled => ({
     ...styles.picker(isDisabled),
     inputAndroid: {
         ...styles.picker(isDisabled).inputNative,
-        paddingLeft: 12,
+        paddingLeft: 0,
     },
 });
```

Comment:
Let's remove all the `index.android` and `index.ios.js` and `index.js` files and move the `pickerstyles` to styles.js

TextIndent will be removed from `index.js` as per my comment above so there won't be any need to create platform files.

Reply:
This sounds like a good plan 👍